README.md

```diff
@@ -6,6 +6,7 @@
 npm install
 npm run dev
 ```
+
```

Comment:
When committing a new change, a ton of changes would appear suddenly, with only the README one with a change, with this empty line, discarding this change will cause the same problem again and again, after committing this, that problem didn't happen again.